selfdrive/locationd/calibrationd.py

```diff
@@ -24,9 +24,11 @@
 MAX_YAW_RATE_FILTER = np.radians(2)  # per second
 
 # This is all 20Hz, blocks needed for efficiency
-BLOCK_SIZE = 100
+SMOOTH_CYCLES = 400  # 20 seconds
```

Comment:
use DT_MDL from common.realtime instead of hardcoding 20Hz

Reply:
Doesn't really depend on time as much as model cycles. Just removed references to time.